mesonbuild/dependencies/mpi.py

```diff
@@ -157,6 +157,10 @@ class IntelMPIConfigToolDependency(_MPIConfigToolDependency):
     """Wrapper around Intel's mpiicc and friends."""
 
     version_arg = '-v'  # --version is not the same as -v
+    # ifort.cfg with: -Xlinker -rpath=... breaks -v (as ifort -v return 1 as exitcode)
+    os.environ["ICCCFG"] = "/dev/null"
```

Comment:
Wouldn't this not work on Windows for instance?

Reply:
the ConfigToolDependency class has some code for handling for tools that return non 0 on success, which... it looks like we actually do but set it to 3?

Reply:
We should probably not be setting global environment variables, but perhaps teaching the ConfigToolDependency base class to recognize a class attribute dict that can be passed into tool invocations as a custom Popen environment.